packages/containers/src/ConfirmDialog/ConfirmDialog.container.js

```diff
@@ -30,12 +31,9 @@ class ConfirmDialog extends React.Component {
 		// and adding onClick event using the retrieved  action creator
 		state.validateAction = getActionsProps(this.context, state.validateAction, state.model);
 		state.cancelAction = getActionsProps(this.context, state.cancelAction, state.model);
+		const props = { ...omit(this.props, cmfConnect.INJECTED_PROPS), ...state };
```

Comment:
remove line 32 and 33
```javascript
const props = { ...omit(this.props, cmfConnect.INJECTED_PROPS), ...omit(state, ['validateAction', 'cancelAction']) };
```

Reply:
as discussed, we're keeping them for now